spec/features/user_index_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe 'user_index', type: :feature do
+  before(:each) do
+    visit root_path
+  end
+
+  it 'shows the username of the the users' do
+    expect(page).to have_content('Bill')
+  end
```

Comment:
- [ ] As I mentioned in the previous comment, you should create users on the user_index_spec page so that you are able to add featured tests for the User index page. You expect that the page has a username `Bill`, even if there is no user with the name `Bill` in your DB.

<img width="491" alt="image" src="https://user-images.githubusercontent.com/77190203/187891180-55163226-ba21-44a0-b8ba-d80b9f05886a.png">

Reply:
Done ✔